R/init.R

```diff
@@ -153,6 +153,16 @@ init <- function(data,
   # log
   teal.logger::log_system_info()
 
+  if (!identical(Sys.getenv("TESTTHAT"), "true")) {
```

Comment:
I can see a few occurence of this. Please consider a new util function to increase readability

Reply:
For TESTHAT there is one. There are 2 occureences for ShinyMockedSession, but the if statement is a one liner. I will leave it as it is.

Reply:
ok